components/schema-migrator/migrations/director/20220106093539_ord-global-registry.up.sql

```diff
@@ -0,0 +1,95 @@
+BEGIN;
+
+ALTER TABLE vendors ALTER COLUMN app_id DROP NOT NULL;
+ALTER TABLE products ALTER COLUMN app_id DROP NOT NULL;
+
+ALTER TABLE products DROP CONSTRAINT products_vendor_fk;
+ALTER TABLE packages DROP CONSTRAINT packages_vendor_fk;
+
+ALTER TABLE vendors DROP CONSTRAINT vendor_ord_id_unique;
+ALTER TABLE vendors ADD CONSTRAINT vendor_ord_id_unique UNIQUE (ord_id);
+
+ALTER TABLE products DROP CONSTRAINT product_ord_id_unique;
+ALTER TABLE products ADD CONSTRAINT product_ord_id_unique UNIQUE (ord_id);
+
+ALTER TABLE products ADD CONSTRAINT products_vendor_fk FOREIGN KEY (vendor) REFERENCES vendors(ord_id);
+ALTER TABLE packages ADD CONSTRAINT packages_vendor_fk FOREIGN KEY (vendor) REFERENCES vendors(ord_id);
+
+------
+
+DROP VIEW api_product;
+
+CREATE VIEW api_product AS
+SELECT api_definitions.id     AS api_definition_id,
+       p.id                   AS product_id
+FROM api_definitions,
+     jsonb_array_elements_text(api_definitions.part_of_products) AS elements
+         JOIN products p ON elements.value = p.ord_id WHERE p.app_id = api_definitions.app_id OR p.app_id IS NULL;
+
+DROP VIEW event_product;
+
+------
```

Comment:
very minor: these `-----` comment separators seems like are placed to group api_product, event_product,  package_product so that it's more readable, but currently the DELETE VIEW of the event_product and package_product overlap with the previous group